{{ cookiecutter.project_slug }}/devops/stacks/plone.yml

```diff
@@ -97,15 +97,30 @@ services:
         - traefik.enable=true
         - traefik.docker.network=traefik-public
         - traefik.constraint-label=traefik-public
+        # AUTHENTICATION
+        # Note: all dollar signs in the hash need to be doubled for escaping.
+        # To create user:password pair, it's possible to use this command:
+        # echo $(htpasswd -nb user password) | sed -e s/\\$/\\$\\$/g
+        # Defaults to admin:admin
+        - traefik.http.middlewares.backend-auth.basicauth.users=admin:$$apr1$$uZPT5Fgu$$AmlIdamxT5ipBvPlsdfD70
```

Comment:
I would use an environment variable for the pw ash instead of hard-coding it in here.

Reply:
Makes sense, but I won't have time for it :( I'm fixing things in a hurry whilst I'm working on the training docs.

Should I remove the ClassicUI and the basic passwd thing in the meanwhile? The other fix is quite important... The rewrite is not done well without it.